templates/CRM/common/standalone.tpl

```diff
@@ -60,6 +56,19 @@
         {include file="CRM/common/footer.tpl"}
       {/if}
     {/crmRegion}
+
+    {* This has to come at the bottom because the variable may not be populated until some of the templates evaluated inline above get evaluated. *}
+    {if $standaloneErrors}
+      <div class="standalone-errors">
+        <ul>{$standaloneErrors}</ul>
+      </div>
+      {literal}
+      <script type="text/javascript">
+        CRM.$("div.standalone-errors").appendTo("div#standalone-errors-wrapper");
```

Comment:
Instead of relying on that wrapper, could just appendTo `#crm-container`.

Reply:
Ok I updated. The spacing is a little odd when using crm-container and before it was under the breadcrumb which has better spacing. Since regular users don't see this it probably doesn't matter but I made it use the breadcrumb when present.